install/app-chrome.sh

```diff
@@ -1,5 +1,9 @@
 cd ~/Downloads
-wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-sudo apt install -y ./google-chrome-stable_current_amd64.deb
-rm google-chrome-stable_current_amd64.deb
-cd -
+if [[ $ARCH == "amd64" ]]; then
+  wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+  sudo apt install -y ./google-chrome-stable_current_amd64.deb
+  rm google-chrome-stable_current_amd64.deb
+elif [[ $ARCH == "arm64" ]]; then
```

Comment:
chromium has not widevine, thereofre cannot play netflix, spotify etc.
Also it might require certain flags for launching with gpu acceleration suuport to watch youtube properly. (guess the latter can only be properly tested on the actual device)
For widevine people often pull some bins from chromeos projects

Reply:
After some recent reserach on this, there is still no binary package with widevine for ubuntu or an arm browser which ships with it.

AsahiLinux is hosting a[ widevine installer ](https://github.com/AsahiLinux/widevine-installer/)script from crhomeos project at googleapis.com. 

And Brave browser published a [manual](https://support.brave.com/hc/en-us/articles/23881756488717-How-do-I-enable-Widevine-DRM-on-Linux) how to link it [#28903](https://github.com/brave/brave-browser/issues/28903).

Chrome has released an arm64 version for windows. Guess a an arm chrome with widevine support is also likely to happen in future. But for now, probably going asahilinux way is best: adding custom installer, updater via chromeos binaries. Probably its best done in another PR once arm support ist merged